src/test/java/HomeWork/AddRemoveElements.java

```diff
@@ -0,0 +1,21 @@
+package HomeWork;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import java.util.List;
+
+public class AddRemoveElements extends PreCondition {
+
+    @Test
+    public void addRemoveElements() {
+        driver.navigate().to("https://the-internet.herokuapp.com/add_remove_elements/");
+        driver.findElement(By.xpath("//button[text()='Add Element']")).click();
+        driver.findElement(By.xpath("//button[text()='Add Element']")).click();
+        driver.findElement(By.xpath("//button[text()='Delete']")).click();
+        List<WebElement> Rows = driver.findElements(By.xpath("//button[text()='Delete']"));
```

Comment:
Why "Rows"? 
And variable name in java should start with small letter, not capital